### ISSUE TITLE
Better "live" error messages

### ISSUE THREAD

**Issue by mutantmonkey** [wuvt/wuvt-site#209]
Currently, errors are not handled very well when loading pages with AJAX. This functionality should be improved.

**Comment by echarlie** [comment]
since very few pages involve AJAX now, I believe this is outdated. Closing.

**Comment by mutantmonkey** [comment]
Nope, all page loads are via AJAX to prevent the stream from stopping when you change pages.
